### ISSUE TITLE
Home Assistant EOF Using new login method

### ISSUE THREAD

**Issue by Lawdie** [flobz/psa_car_controller#775]
**Describe the bug**
When following the new instructions to login you are unable to input your OAUTH return code for section 6 due to the script failing when you press "submit" within PSACC

**To Reproduce**
1. Attached myself to the PSACC docker container from within home assistant
2. Followed instructions shown on the configuration page, when you press submit to generate the external link it EOF's
3. You can paste the code into the terminal however OTP fails and submit will just generate another error

**Environment (please complete the following information):**
 - OS: Homeassistant OS
 - Vauxhall Corsa E 
 - Application version 3.4.0

Error from docker container &  HA log when pressing submit 

```
2024-02-24 22:55:13,252 DEBUG https://mw-vx-m2c.mym.awsmpsa.com:443 "POST /api/v1/user?culture=en_GB&width=1080&version=1.33.0 HTTP/1.1" 200 None
2024-02-24 22:55:13,259 :: INFO :: Now login to this URL in a browser: https://idpcvs.vauxhall.co.uk/am/oauth2/authorize?client_id=CLIOENTID&redirect_uri=mymvxsdk%3A%2F%2Foauth2redirect%2Fgb&response_type=code&scope=openid%20profile&state=STATE&code_challenge=CHALLENGE&code_challenge_method=S256

Copy+paste the resulting mymXX-code (in F12 > Network, when you hit the final OK button, 36 chars, UUID format): 2024-02-24 22:55:13,261 :: ERROR :: EOF when reading a line
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/psa_car_controller/web/view/config_views.py", line 139, in connectPSA
    res = firstLaunchConfig(app_name, email, password, countrycode)
  File "/usr/local/lib/python3.9/dist-packages/psa_car_controller/psa/setup/app_decoder.py", line 95, in firstLaunchConfig
    psacc.connect()
  File "/usr/local/lib/python3.9/dist-packages/psa_car_controller/psacc/application/psa_client.py", line 35, in connect
    self.manager.init_with_oauth2_redirect(realm_info[self.realm]["scheme"], self.country_code)
  File "/usr/local/lib/python3.9/dist-packages/psa_car_controller/psa/oauth.py", line 49, in init_with_oauth2_redirect
    ret = input("\nCopy+paste the resulting mymXX-code (in F12 > Network, "
EOFError: EOF when reading a line
```

**Comment by frans0180** [comment]
Hallo.  I have the same problem 

EOF.    See at the bottom of the picture 
And the log file is very very long ! 

![IMG_9747](https://github.com/flobz/psa_car_controller/assets/150475372/171f35ed-66f4-4f9f-acc1-1dbc6d69cbe3)

**Comment by nicknol** [comment]
same here for a Citroen eC4

**Comment by 290989** [comment]
Same here for a Peugeot E-208.

**Comment by Arnoudd** [comment]
Same here. Also with a Peugeot e-208.

**Comment by adambogocz** [comment]
It has nothing to do with car model, check here for more info https://github.com/flobz/psa_car_controller/issues/777#issuecomment-1963771810

**Comment by flobz** [comment]
fix in version v3.4.1

**Comment by 290989** [comment]
Do you will update psacc-ha, too?
